.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,60 @@
+## Summary of Changes
+
+Please write a sentence or two summarizing the proposed change.
+
+Addresses [ORCA-XX: Develop amazing new feature](https://bugs.earthdata.nasa.gov/browse/ORCA-XXX)
+
+## Changes
+
+* Detailed list or prose of changes
+* ...
+
+## Type of change
+
+Please delete options that are not relevant.
+
+- [ ] Bug fix (non-breaking change which fixes an issue)
+- [ ] New feature (non-breaking change which adds functionality)
+- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
+- [ ] This change requires a documentation update
+
+## How Has This Been Tested?
+
+Please describe the tests that you ran to verify your changes. Provide instructions so we can reproduce. Please also list any relevant details for your test configuration
+
+- [ ] Test A
+- [ ] Test B
+
+**Test Configuration**:
+* Firmware version:
+* Hardware:
+* Toolchain:
+* SDK:
+
+## Checklist:
+
+- [ ] My code follows the style guidelines of this project
+- [ ] I have performed a self-review of my own code
+- [ ] I have commented my code, particularly in hard-to-understand areas
+- [ ] I have made corresponding changes to the documentation
```

Comment:
Possibly expand to call out specific documentation they should always update. Maybe something line the following:
```
- [ ] I have made corresponding changes to the documentation including but not limited to
    - [] CHANGELOG.md
    - [] Testing documentation
    - [] Development documentation
    - [] User documentation
```

Reply:
```suggestion
- [ ] I have made corresponding changes to the necessary documentation (remove those that do not apply)
    - [] CHANGELOG.md
    - [] Testing documentation
    - [] Development documentation
    - [] User documentation
    - [] (Additional here)
```

Reply:
@bhazuka what do you think about this?